app/src/main/java/com/example/dine_and_donate/EventActivity.java

```diff
@@ -42,20 +56,29 @@
     private Spinner mEndHour;
     private Spinner mEndMin;
     private Spinner mEndHalf;
-
     private EditText mEtEventInfo;
     private Button mBtnCreate;
+    private Button mChooseImage;
+    private ImageView ivVoucher;
```

Comment:
hmm let's still use m here. mIvVoucher is fine, or mVoucherImageView, either way